mandatory/1-fix-functions.js

```diff
@@ -21,7 +20,7 @@ function mood() {
 }
 
 function greaterThan10(num) {
-  let isBigEnough;
+  let isBigEnough = num > 10;
```

Comment:
Looks good! Another way to do it would be to remove the isBigEnough variable and just have "if (num > 10)" but functionally it's identical so it only depends on what you think is more clear to other people!